nautilus_trader/core/correctness.pyx

```diff
@@ -1201,13 +1200,13 @@ class PyCondition:
         Condition.positive(value, param, ex_type)
 
     @staticmethod
-    def positive_int(int64_t value, str param, ex_type = None):
+    def positive_int(value: int, str param, ex_type = None):
```

Comment:
Relaxed all of these C types, as they're coming from Python in non-performance critical function calls.
Avoids having to name the specific integer types in C.